Sources/Interaction/Widgets/OrientationMarkerWidget/index.d.ts

```diff
@@ -147,6 +147,13 @@ export interface vtkOrientationMarkerWidget extends vtkObject {
 	 * Updates the orientation widget viewport size.
 	 */
 	updateViewport(): void;
+
+	/**
+	 * An instance of this class will spawn its own renderer, by default non interactive.
+	 * This behavior is configurable through the spawnInteractiveRenderer property when initializing the instance.
+	 * @returns true if the renderer was created as interactive, false otherwise.
+	 */
+	getSpawnInteractiveRenderer(): boolean;
```

Comment:
you also need to add it to `IOrientationMarkerWidgetInitialValues`

Reply:
Rename into `interactiveRenderer` or `interactive`?

Reply:
I thought you would also spawn `vtkInteractiveOrientationWidget` within `OrientationMarkerWidget` when `interactive` is true...